### PR TITLE
The __future__ is now

### LINKED ISSUE #208: Obsolete __future__ imports
spotted during CR of #207

### PR DESCRIPTION
**Pull request by lostanlen**
remove `__future__` imports, alphabetize imports

functionally equivalent to base branch

triaged for v0.2?

closes #208

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -130,16 +130,13 @@ For more details, please visit: [website]
...
-from __future__ import division
-from __future__ import print_function
 
 # -- import whatever you need here
 
-import mirdata.track as track
-import mirdata.utils as utils
 import mirdata.download_utils as download_utils
 import mirdata.jams_utils as jams_utils
+import mirdata.track as track
```
Comment (rabitt) [inline CONTRIBUTING.md:138]: while we're here - should we maybe standardize all these style imports to `from mirdata import track` and `from mirdata import utils`?

Comment (rabitt) [reply CONTRIBUTING.md:138]: Just saw you're using this convention below - let's change it here in the CONTRIBUTING too!

Comment (lostanlen) [reply CONTRIBUTING.md:138]: Noted

Comment (lostanlen) [reply CONTRIBUTING.md]: fixed in a7fe72e

**Comment by lostanlen** [comment]
Thank you @rabitt for the review. I standardized the import style.
Merging

**Pull request merged**